kobo/apps/help/models.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# 😇
+from __future__ import unicode_literals
+
+import datetime
```

Comment:
Same comment on imports :-)

Reply:
The grouping appears to follow PEP-8's recommendations already?
```python
# `__future__` must come first, right?
from __future__ import unicode_literals

# Standard library imports.
import datetime

# Related third party imports.
from django.db import models
from django.conf import settings
from markdownx.models import MarkdownxField
from private_storage.fields import PrivateFileField
from django.utils.module_loading import import_string
from jsonbfield.fields import JSONField
from markdownx.settings import MARKDOWNX_MARKDOWNIFY_FUNCTION

# Local application/library specific imports.
from kpi.fields import KpiUidField
```

My sorting of "Related third party imports" here certainly leaves something to be desired. Our old [code style document](https://docs.google.com/document/d/17j7DJh79l1GItIZyFGfhqLTg-doj0g1MbNBDxsvoyko/edit) mentions sorting within each PEP-8 section by [line length](https://github.com/timothycrosley/isort#ordering-by-import-length), and it looks like I was doing that but misplaced `from jsonbfield.fields import JSONField`. I'd prefer the line-length approach over alphabetical sorting, as it's more readable to me.